eng/pipelines/release-cli.yml

```diff
@@ -3,7 +3,7 @@ resources:
     - repository: azure-sdk-build-tools
       type: git
       name: internal/azure-sdk-build-tools
-      ref: refs/tags/azure-sdk-build-tools_20230828.2
+      ref: refs/tags/azure-sdk-build-tools_20230829.1
```

Comment:
This should take on whatever value is in `main`. 

Provided your local `main` branch is up to date this can be done by running: 

```bash 
git checkout main -- eng/pipelines/release-cli.yml
```

Reply:
I didn't make this change - likely a rebase/timing issue.